bedrock/mozorg/templates/mozorg/contribute/friends.html

```diff
@@ -14,55 +14,95 @@
   {% stylesheet 'contribute-friends' %}
 {% endblock %}
 
-{% block js %}
-{% endblock %}
-
 {% block contrib_page_title %}{{ _('Join Firefox Friends today!') }}{% endblock %}
 
 {% block alt_header %}
-<img src="{{ static('img/contribute/friends/firefox-friends-logo.svg') }}" width="186" class="ff-logo" data-png="{{ static('img/contribute/friends/firefox-friends-logo.png') }}" data-fallback="true" alt="{{_('Firefox Friends logo')}}" />
+<img src="{{ static('img/contribute/friends/firefox-friends-logo.svg') }}" width="186" class="ff-logo" data-png="{{ static('img/contribute/friends/firefox-friends-logo.png') }}" data-fallback="true" alt="{{ _('Firefox Friends logo') }}" />
 <div class="ff-header-content">
-  <h1>{{_('Join Firefox Friends today!')}}</h1>
```

Comment:
This page is localized in a few languages, you can't change strings like this, you need a tag.